app/models/account_authorization_config/clever.rb

```diff
@@ -64,7 +65,17 @@ def provider_attributes(token)
   protected
 
   def me(token)
-    token.options[:me] ||= token.get("/me").parsed['data']
+    token.options[:me] ||= begin
+                             raw_data = token.get("/v2.1/me").parsed
+                             data = raw_data["data"].dup
+                             data = data.merge(token.get("/v2.1/#{raw_data["type"]}s/#{data["id"]}").parsed["data"])
+                             data["name.first"] = data.dig("name", "first")
+                             data["name.last"] = data.dig("name", "last")
+                             data['name.full'] = "#{data['name.first']} #{data['name.last']}"
+                             data["district_username"] = data.dig("credentials", "district_username")
+                             data.slice!(*(self.class.recognized_federated_attributes + ["district"]))
+                             data
```

Comment:
i think we want a `rescue` here and we also probably want to raise a sentry exception inside the rescue if we raise an exception.

Reply:
Updated to add a rescue and sentry exception handling